internal/dic/dic.go

```diff
@@ -131,7 +131,7 @@ func Load(path string) (d *Dic, err error) {
 			if e != nil {
 				return e
 			}
-			rc.Close()
+			defer rc.Close()
```

Comment:
__[gometalinter]__
* [errcheck] error return value not checked (defer rc.Close())

<!-- Comment by [SideCI](https://sideci.com) -->